src/stim/diagram/timeline/timeline_svg_drawer.cc

```diff
@@ -786,14 +786,32 @@ void DiagramTimelineSvgDrawer::make_diagram_write_to(
         }, 24);
     }
 
-    // Make sure qubit lines are drawn first, so they are in the background.
+    // Make sure qubit lines/points are drawn first, so they are in the background.
     if (mode != SVG_MODE_TIMELINE) {
         // Draw qubit points.
+        auto tick = tick_slice_start;
         for (uint64_t col = 0; col < obj.num_cols; col++) {
             for (uint64_t row = 0; row < obj.num_rows && row * obj.num_cols + col < tick_slice_num; row++) {
                 for (auto q : obj.detector_slice_set.used_qubits()) {
-                    auto c = obj.coord_sys.qubit_coords[q];
+                    std::stringstream id_ss;
+                    id_ss << "qubitdot";
```

Comment:
Bit worried about this detail-heavy logic being repeated in two places. Extract a method?